src/buffer/out/ut_textbuffer/TextBuffer.Unit.Tests.vcxproj

```diff
@@ -18,6 +19,9 @@
     </ClCompile>
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\..\types\lib\types.vcxproj">

```

Comment:
Do we need to add `types` to the `TARGETLIBS` for this test too?

Reply:
GOOD CATCH.

Reply:
fixed!